internal/datanode/meta_service.go

```diff
@@ -49,28 +49,11 @@ func newMetaService(rc types.RootCoord, collectionID UniqueID) *metaService {
 // TODO: Replace with getCollectionInfo below.
 // getCollectionSchema get collection schema with provided collection id at specified timestamp.
 func (mService *metaService) getCollectionSchema(ctx context.Context, collID UniqueID, timestamp Timestamp) (*schemapb.CollectionSchema, error) {
-	req := &milvuspb.DescribeCollectionRequest{
-		Base: &commonpb.MsgBase{
-			MsgType:   commonpb.MsgType_DescribeCollection,
-			MsgID:     0, //GOOSE TODO
-			Timestamp: 0, // GOOSE TODO
-			SourceID:  Params.DataNodeCfg.GetNodeID(),
-		},
-		DbName:       "default", // GOOSE TODO
-		CollectionID: collID,
-		TimeStamp:    timestamp,
+	response, err := mService.getCollectionInfo(ctx, collID, timestamp)
+	if response != nil {
+		return response.GetSchema(), err
```

Comment:
why remove the check of ErrorCode()?

Reply:
Before modification, `getCollectionSchema` and `getCollectionInfo` were the same. The check of ErrorCode() has been included in the `getCollectionInfo`.